docs/sync-storage-handbook/src/ffi.md

```diff
@@ -0,0 +1,3 @@
+# FFI
+
+All libraries expose a C-compatible [foreign function interface](https://docs.rs/ffi-support/0.1.3/ffi_support/) for platform-specific bindings.
```

Comment:
@thomcc I recall you having a better phrasing for this than "C-compatible" but I don't recall what it was.  Maybe it was just the more specific "C-ABI-compatible"?